app/src/main/java/com/swentseekr/seekr/ui/profile/ProfileViewModel.kt

```
@@ -42,37 +45,95 @@ class ProfileViewModel(
   val currentUid: String?
     get() = injectedCurrentUid ?: firebaseAuth?.currentUser?.uid
 
+  private val _totalReviews = MutableStateFlow(0)
+  val totalReviews: StateFlow<Int> = _totalReviews.asStateFlow()
+
+  fun loadTotalReviewsForProfile(profile: Profile) {
+    viewModelScope.launch {
+      var total = 0
+      profile.myHunts.forEach { hunt ->
+        try {
+          val reviews = HuntReviewRepositoryProvider.repository.getHuntReviews(hunt.uid)
+          total += reviews.size
+        } catch (e: Exception) {}
```

There is a sonar cloud issue link to this because the empty catches swallow errors silently. If a repository call fails, you’ll never know why. At least put a Log in there.